### ISSUE TITLE
A11y_VSCode_Debug Console_Keyboard: Clear console button is not accessible through keyboard as the data is not cleared from debug console pane when screenreader is on

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#99007]
[Check out Accessibility Insights!](https://github.com/microsoft/AzureStorageExplorer/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS2.1.1;#Win10;#Visual Studio Code;#NVDA;#JAWS;#Keyboard; #CAITestPass_May2020;#FTP;#STP;

**Environment Details:**
Application Name: Visual Studio Code
Windows Version: Windows10
Screenreader version:NVDA[2020.1], JAWS 2020.2001.70.400
**Repro Steps:**
1) Open Visual studio code   editor 
2) Press ctrl+shift+y 
3)Validate it's possible to clear the debug console output when NVDA is ON.

**Actual:**
While navigating through keyboard,Clear console button is not accessible through keyboard as the data is not cleared from debug console pane when screenreader is on.
This issue is reproduced with NVDA and Jaws.

**Expected:**
Clear console button should be accessible through keyboard when screenreader is on.

**Attachment link for Reference:**
https://microsoft-my.sharepoint.com/:v:/p/v-nisaho/EUoxdD7KrexEp__yZ6UkEUcBpI0YvKJKteLO172v34W7OA?e=D84KCJ

**Comment by isidorn** [comment]
You can focus the clear console by 
Ctrl + Shift + Y, press shit + Tab a couple of times and the focus is on the action. You might need to first F1 > tab key to move focus, but that is expected since you are in the editor.
You can also F1 > clear debug console, you can aslo assign a custom keybinding to it to make it faster.

Thus closing as designed
